doc/source/upgrading.md

```diff
@@ -26,7 +26,7 @@ These steps are **critical** before performing a major upgrade.
 
 ### v0.5 to v0.6
 
-See the [CHANGELOG](https://github.com/jupyterhub/zero-to-jupyterhub-k8s/blob/d1b2804d887033f5d0e1f5315cfe8d3c7bbf41d2/CHANGELOG.md#06---ellyse-perry---2017-01-29).
+See the [CHANGELOG](https://github.com/jupyterhub/zero-to-jupyterhub-k8s/blob/master/CHANGELOG.html#06---ellyse-perry---2017-01-29).
```

Comment:
I don't believe that this file exists in `html` form...did you mean to link to the docs?